unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -224,6 +221,8 @@ public void Cleanup(bool immediate)
             if (isReleased)
                 return;
 
+            DisposeAllSceneComponents();
```

Comment:
I think we should make each entity clean itself in `ParcelScenesCleaner` instead of this scene-wide approach, note the benchmarking comment.

Reply:
Done @BrianAmadori, please take a look.